packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -55,6 +55,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint256 minAgreementRequirement; // Minimum staked tokens that must agree on an outcome to resolve a request.
         uint256 cumulativeStakeAtRound; // Total staked tokens at the start of the round.
         uint64 resolvedIndex; // Index of pendingPriceRequestsIds that has been traversed this round.
+        uint32 requestToVoteOnInThisRound; // The number of requests that have been voted on in this round.
```

Comment:
Maybe the name of the variable could be more explicit with something like `numVotedRequests` or `numRequestsToVote`